exercises/practice/connect/src/main/java/Connect.java

```diff
@@ -0,0 +1,16 @@
+class Connect {
+
+    public Connect(String[] board) {
+        throw new UnsupportedOperationException("Implement this function");
+    }
+
+    public Winner computeWinner(String[] board) {
+        throw new UnsupportedOperationException("Implement this function");
+    }
+
+    enum Winner {
+        PLAYER_X,
+        PLAYER_O,
+        NONE
+    }
```

Comment:
This enum should be in a different file that the online editor displays but does not allow edits for.